From adf16fdda289c862f0682f2a68ece138f9722f9f Mon Sep 17 00:00:00 2001
From: Ahmad Abbas <ahmada@marvell.com>
Date: Thu, 5 May 2016 20:18:26 +0300
Subject: [PATCH 157/239] rfu: a70x0: opened window for ihb 0 and ihb 1

opened window for ihb 0 and ihb 1 in the rfu for a70x0.

Change-Id: Ie8f8bb25d6d273c229197c345897c140a54e963d
Signed-off-by: Ahmad Abbas <ahmada@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29658
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 plat/marvell/a8k/a70x0/board/marvell_plat_config.c | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/plat/marvell/a8k/a70x0/board/marvell_plat_config.c b/plat/marvell/a8k/a70x0/board/marvell_plat_config.c
index d266af2..7c8e441 100644
--- a/plat/marvell/a8k/a70x0/board/marvell_plat_config.c
+++ b/plat/marvell/a8k/a70x0/board/marvell_plat_config.c
@@ -59,12 +59,13 @@ int marvell_get_amb_memory_map(struct amb_win **win, uint32_t *size)
 /*******************************************************************************
  * RFU Configuration
  ******************************************************************************/
-/*
+
+
 struct rfu_win rfu_memory_map[] = {
-	{0x0,	0xf9100000,	0x0,	0x1f00000,	IHB_0_TID},
+	{0x0,	0xf4000000,	0x0,	0x2000000,	IHB_0_TID},
+	{0x0,	0xf6000000,	0x0,	0x2000000,	IHB_1_TID},
 };
-*/
-struct rfu_win *rfu_memory_map = NULL;
+
 
 uintptr_t marvell_get_rfu_reg_offs(void)
 {
-- 
1.9.1

